torch/_torch_docs.py

```diff
@@ -3098,6 +3098,29 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.i0,
+           r"""
+i0(input, *, out=None) -> Tensor
+
+Computes the modified Bessel function of the first kind, order 0, for each element of :attr:`input`.
```

Comment:
For consistency, this might become "Computes the zeroth order modified Bessel function of the first kind for each element of :attr:\`input\`."